go/engine/login_provision.go

```diff
@@ -338,8 +343,7 @@ func (e *loginProvision) paper(m libkb.MetaContext, device *libkb.Device, keys *
 	// the paper key on the global and not thread-local active device.
 	m.ActiveDevice().CacheProvisioningKey(m, keys)
 
-	e.saveToSecretStore(m)
```

Comment:
This could have been intentional...I can't remember exactly, but there were some situations where we didn't want to block provisioning if there was an error saving to secret store.

Reply:
The fact that it's like this everywhere makes me think that is the case...I would be a little worried to make this change.

Reply:
Cool, to prevent future misunderstandings I'll change that function to not return an error at all.  Maybe it was only doing so it log it in the `defer`, so I'll do that another way.